css/base.css

```diff
@@ -0,0 +1,585 @@
+ * NOTE: the default cannot safely be changed in Chrome and Safari on OS X.
+ */
+
+optgroup {
+  font-weight: bold;
+}
+
+/* Tables
+   ========================================================================== */
+
+/**
+ * Remove most spacing between table cells.
+ */
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+td,
+th {
+  padding: 0;
+}
+/*custom layout*/
+
+/*fonts*/
+@font-face {
+    font-family: 'LatoRegular';
+    src: url('fonts/lato-regular.eot');
+    src: url('fonts/lato-regular.eot') format('embedded-opentype'),
+         url('fonts/lato-regular.woff2') format('woff2'),
+         url('fonts/lato-regular.woff') format('woff'),
+         url('fonts/lato-regular.ttf') format('truetype'),
+         url('fonts/lato-regular.svg#LatoRegular') format('svg');
+}
+@font-face {
+    font-family: 'LatoBoldBold';
+    src: url('fonts/lato-bold.eot');
+    src: url('fonts/lato-bold.eot') format('embedded-opentype'),
+         url('fonts/lato-bold.woff2') format('woff2'),
+         url('fonts/lato-bold.woff') format('woff'),
+         url('fonts/lato-bold.ttf') format('truetype'),
+         url('fonts/lato-bold.svg#LatoBoldBold') format('svg');
+}
+@font-face{
+    font-family: 'Roboto';
+     src: url('fonts/RobotoMono-Regular.ttf') format('truetype');
+     font-weight: normal;
+     font-style: normal;
+   }
+/*Using Iconmoon fonts*/
+@font-face {
+  font-family: 'icomoon';
+  src:url('../fonts/icomoon.eot?-tzpryb');
+  src:url('../fonts/icomoon.eot?#iefix-tzpryb') format('embedded-opentype'),
+    url('../fonts/icomoon.ttf?-tzpryb') format('truetype'),
+    url('../fonts/icomoon.woff?-tzpryb') format('woff'),
+    url('../fonts/icomoon.svg?-tzpryb#icomoon') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  font-family: 'icomoon';
+  speak: none;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-home:before {
+  content: "\e900";
+}
+.icon-keyboard:before {
+  content: "\e955";
+}
+.icon-arrow-up:before {
+  content: "\ea32";
+}
+.icon-github:before {
+  content: "\eab1";
+}
+
+
+
+/*main styles*/
+
+/*phones*/
+@media(min-width: 300px){
+  p{
+    font-size: 0.938em;
+  }
+
+}
+/*tablets*/
+@media(min-width: 768px){
+  header{
+    height: 6.250em;
+  }
+  body{
+    height: 50%;
+  }
+  p{
+    font-size: .625em;
+  }
+
+}
+/*Desktop*/
+@media(min-width: 1200px){
+
+  h1{
+    font-size: 1.618em;
+    padding: 0.313em;
+  }
+  h2{
+    font-size: 1.618em;
+    font-family: 'LatoBoldBold';
+    text-align: center;
+    margin: 2px;
+  }
+  nav{
+    display: fix;
+    font-family: 'LatoRegular';
+    margin: 2.500em;
+  }
+  input{
+    border-radius: 0.625em;
+  }
+  p{
+    font-size: 25px;
+  }
+  article{
+    height: 350px;
```

Comment:
There should not be a height declaration here. A element's height should be defined by its content.